rust/chains/abacus-ethereum/src/inbox.rs

```diff
@@ -233,7 +233,7 @@ where
         Ok(self.contract.remote_domain().call().await?)
     }
 
-    #[tracing::instrument(err)]
+    #[tracing::instrument(err, skip(proof))]
```

Comment:
should this also skip self?

Reply:
Yeah so I was thinking about skipping self here, but i have found myself finding it quite useful at times to identify the inbox, signer address, etc. so I was inclined to keep it for now. I do think it should really be kept in a different span ideally